### PR TITLE
Upload files with size more than 32MB

### PR DESCRIPTION
**Pull request by Grandman**
Hello!
I think it is an important feature (at least for me)
Made according with API docs(https://developers.virustotal.com/v3.0/reference#files-upload-url)
I moved logic to separate method, because it double requests for upload (get upload_url and upload request) and free version limit is a 4 requests per minute.

### CODE REVIEW

**Comment by pwelch** [comment]
Hey @Grandman Thanks for the PR! 

Looks like there's an issue with tests. Can you fix that up? 

Thanks!

File: README.md [hunk 1]
```diff
@@ -65,7 +65,10 @@ require 'virustotal_api'
 file    = '/path/to/file'
 api_key = 'MY_API_KEY'
 
+# for upload file
 vtscan = VirustotalAPI::File.upload(file, api_key)
+# or big file (more than 32MB)
+vtscan = VirustotalAPI::File.upload_big(file, api_key)
```
Comment (pwelch) [inline README.md:71]: @Grandman can we rename this to: `.upload_large` ?

Comment (Grandman) [reply README.md]: Yes of course

**Comment by Grandman** [comment]
fixed

**Comment by pwelch** [comment]
@Grandman Thanks! I'll merge this and create a release later this week

**Comment by pwelch** [comment]
@Grandman these changes have been released in [v0.5.5](https://rubygems.org/gems/virustotal_api/versions/0.5.5). Thanks for your changes!

**Pull request merged**